src/components/ProviderCard/ProviderCard.stories.js

```diff
@@ -0,0 +1,24 @@
+import ProviderCard from './ProviderCard';
+import DIDLogo from '../../assets/logos/didperu.svg';
+
+export default {
+  title: 'Pages/Providers/ProviderCard',
+  component: ProviderCard,
+};
+
+// More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
+const Template = (args) => ({
+  components: { ProviderCard },
+  setup() {
+    return { args };
+  },
+  template:
+    '<div style="justify-content: center; background-color: #ccc; display: flex; padding: 10px;"><provider-card v-bind="args" /></div>',
+});
```

Comment:
Can you move this style to a CSS or at least to a "style={{}}" kind of object, or maybe a string?

I am kinda nervous to see it hardcoded

Reply:
Of course! Will do